core/object.cpp

```diff
@@ -948,16 +948,6 @@ String Object::to_string() {
 void Object::_changed_callback(Object *p_changed, const char *p_prop) {
 }
 
-void Object::add_change_receptor(Object *p_receptor) {
```

Comment:
This is fine, just leave Object as-is, because this is used by editor and will eventually be refactored.